lib/SIL/SIL.cpp

```diff
@@ -0,0 +1,11 @@
+//===--- SIL.cpp ---------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
```

Comment:
Do we really need this? If we do, can we put a fat comment here telling people /not/ to put anything here?

Reply:
Yeah, we can totally put in some comments to explain that in both the files.